package.json

```diff
@@ -125,7 +125,7 @@
     "singleQuote": true
   },
   "engines": {
-    "node": "10.x",
+    "node": "13.x",
```

Comment:
should probably be more flexible with this and change to  >=

Reply:
What do you mean by this? to do `>= 10.x` or something else?

Reply:
Ah, nevermind, I misunderstood - this is why I shouldn't review PRs first thing after waking up :P LGTM!